prettier.py

```diff
@@ -1,7 +1,10 @@
-from subprocess import Popen, check_output, CalledProcessError, PIPE
+import subprocess
+import platform
```

Comment:
I don't why this is shown as a change, it's already like that in your repo: https://github.com/danreeves/sublime-prettier/blob/master/prettier.py